File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        for i in range(length):
            for j in range(length):
                if nums[i]+nums[j] == target and i != j:
                    result = [i,j]
                    return result
        

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> h = new HashMap<>();
        for(int i =0; i<nums.length; i++){
            int k = target-nums[i];
            if(!h.containsKey(k)){
                h.put(nums[i],i );
            }
            else{
                return new int[]{h.get(k),i};
            }
        }
        return new int[]{-1,-1};
    }
}